Framework/PythonInterface/mantid/api/src/Exports/DetectorInfo.cpp

```diff
@@ -0,0 +1,14 @@
+#include "MantidAPI/DetectorInfo.h"
+#include <boost/python/class.hpp>
+
+using Mantid::API::DetectorInfo;
+using namespace boost::python;
+
+void export_DetectorInfo() {
+  class_<DetectorInfo, boost::noncopyable>("DetectorInfo", no_init)
+      .def("size", &DetectorInfo::size, (arg("self")),
```

Comment:
Sorry, I didn't check the current state of `DetectorInfo` and thought the question was regarding clashing names. I think `size()` is fine here too. Should we add `__len__` too so that `len()` works?